Example/SmileID/HomeViewModel.swift

```diff
@@ -86,6 +86,12 @@ class HomeViewModel: ObservableObject, SmartSelfieResultDelegate, DocumentCaptur
                                                object: nil)
     }
 
+    func didSucceed(documentFrontImage: Data,
```

Comment:
We should also include the selfie file here, since that's another image that is captured